lib/Dependency.js

```diff
@@ -7,7 +7,7 @@ function Dependency() {
 }
 module.exports = Dependency;
 
-Dependency.prototype.isEqualResource = function( /* other */ ) {
```

Comment:
It's sad that this has to be removed. It was intended as documentation of the available parameters